appveyor.yml

```diff
@@ -0,0 +1,94 @@
+environment: 
+    matrix:
+        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+          DB: 10.3.11
+          CMAKE_PARAM_G: 'Visual Studio 14 2015 Win64'
+
+        - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+          DB: 10.3.11
+          CMAKE_PARAM_G: 'Visual Studio 15 2017 Win64'
+
+# scripts that are called at very beginning, before repo cloning
+init:
+    - git config --global core.autocrlf input
+    - wmic cpu get NumberOfCores
+    - wmic ComputerSystem get TotalPhysicalMemory
+
+clone_folder: c:\maodbc
+platform: x64
+configuration: Release
+
+install:
+    
+
+build_script:
+    # build libmariadb separately first because otherwise the Wix installer build might look for files that aren't available yet
+    - cd libmariadb
+    - cmake --build . --config RelWithDebInfo --parallel 2
```

Comment:
There is no need to build libmariadb - c/odbc's cmake does that

Reply:
I know, but there is some build parallelism problem in that case (at least in appveyor). That's why I build libmariadb separately first.